### PR TITLE
code-component. Update all components to new sailor version / use new logger functionality / dockerize

### PR DESCRIPTION
**Pull request by ElBabbler**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "eio-code-component",
-  "version": "1.1.0",
+  "version": "1.1.1",
   "description": "elastic.io code component",
   "engines": {
     "node": "^10"
```
Comment (kirill-levitskiy) [inline package.json]: Sailor 2.6.0 requires nodes 12+

File: actions/code.js [hunk 2]
```diff
@@ -8,10 +8,10 @@ const request = require('co-request');
 function wait(timeout) {
   return new Promise((ok) => {
     setTimeout(() => {
-      this.logger.debug('Done wait');
+      this.logger.info('Done wait');
```
Comment (uaArsen) [inline actions/code.js]: Why changing this? This is existing behavior and out of scope of this task

Comment (ElBabbler) [reply actions/code.js]: Igor asked for this in general components

File: package.json (hunk 1 above)
Comment (ElBabbler) [reply package.json]: Changed

**Pull request merged**